File: c++/Esercizi/GreenPass/persona.cpp
#include <iostream>
using namespace std;

class Persona{
protected:
    string nome;
    string cognome;
    string cf;
public:
    Persona() {
        nome = "xxx";
        cognome = "xxx";
        cf = "xxx";
    }

    Persona(string n, string c, string cf) {
        this->nome = n;
        this->cognome = c;
        this->cf = cf;
    }

    void stampa(ostream& dest) {
        dest << nome << ' ' << cognome << endl;
        dest << "Codice discale: " << cf << endl;
    }
};

File: java/EserciziEdutecnica/src/persona/Persona.java
package persona;

public class Persona {
    private final String nome;
    private final String cognome;
    private final String CF;

    public Persona(String nome, String cognome, String CF){
        this.nome = nome;
        this.cognome = cognome;
        this.CF = CF;
    }

    public String getNome(){
        return nome;
    }

    public String getCognome(){
        return cognome;
    }

    public String getCF(){
        return CF;
    }

    @Override
    public String toString(){
        return nome + " " + cognome + " " + CF;
    }
}
